examples/bert/user_script.py

```diff
@@ -124,13 +125,13 @@ def post_process(output):
 # -------------------------------------------------------------------------
 
 
-@Registry.register_dataset("bert_dataset")
-def create_dataset(data_name: str, split: str, language: str, token: Union[bool, str] = True):
+@Registry.register_dataset()
+def bert_dataset(data_name: str, split: str = None, language: str = None, token: Union[bool, str] = True):
```

Comment:
`split`, `language`, `token` could all be removed. I think they were copy pasted here from the llama example when you made the perf tuning changes last time.

Reply:
Removed.